#!/usr/bin/env python
PACKAGE = "masterslave"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("safetyHeight", double_t, 0, "safety height", 0.05, 0, 2000)
gen.add("apertureLimit", int_t, 0, "aperture of the frustum", 60, 45, 80)
gen.add("gripperVelocity", double_t, 0, "velocity of the gripper", 0.1, 0.01, 100)

exit(gen.generate(PACKAGE, "kinematic", "kinematic"))
